Doc/library/pyexpat.rst

```diff
@@ -614,6 +614,13 @@ otherwise stated.
 
 .. method:: xmlparser.ExternalEntityRefHandler(context, base, systemId, publicId)
 
+   .. warning::
+
+      Registering a handler for external entity references may allow
+      attacker-controller XML to access local files and/or the network,
+      and thus create new security risks.
+      By default, :class:`xmlparser` is safe from these threats.
```

Comment:
I think this warning is misleading. Registering a handler for external entity references only allows access to local files and/or the network if the handler allows unchecked access to local files and/or the network. It is your code a source of vulnerability, not the XML parser.